src/Entity/Customer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Created by solutionDrive GmbH
+ *
+ * @author    lei wang <wang@solutiondrive.de>
+ * @date      03.02.18
```

Comment:
I think you have to remove @date and @time annotations as that makes problems with doctrine's annotations

Reply:
i didnt even know that x)

Reply:
Just try to write an orm and run doctrine:schema:update, it will crash.